src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
-public class MainApp {
+import mashines.Bulldozer;
+import mashines.Excavator;
+import mashines.Truck;
 
+public class MainApp {
+    public static void main(String[] args) {
+        Mashine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
+        for (Mashine mashine: machines) {
```

Comment:
Let's slightly fix indentation issue :)
```suggestion
        for (Mashine mashine : machines) {
```